test/unit/test_type.c

```diff
@@ -1705,6 +1705,146 @@ bool test_path_by_offset_typedef(void) {
 	mu_end;
 }
 
+bool test_offset_by_path_struct(void) {
+	RzTypeDB *typedb = rz_type_db_new();
+	const char *types_dir = TEST_BUILD_TYPES_DIR;
+	rz_type_db_init(typedb, types_dir, "x86", 64, "linux");
+
+	// simple
+
+	// eprintf("=============== SIMPLE ===============\n");
+	char *error_msg = NULL;
+	RzType *ttype = rz_type_parse_string_single(typedb->parser, "struct Hello { int32_t a; uint32_t b; };", &error_msg);
+	mu_assert_notnull(ttype, "type parse successful");
+	mu_assert_eq(ttype->kind, RZ_TYPE_KIND_IDENTIFIER, "parsed type");
+	mu_assert_streq(ttype->identifier.name, "Hello", "parsed type");
+
+	/**
+	 * Set offsets manually.
+	 * Since parse_struct_node() in librz/type/parser/types_parser.c sets them to 0.
+	 */
```

Comment:
Is this behaviour correct? cc @XVilka

Reply:
It was planned to remove offsets completely, because they should be calculated on the fly, depending on the architecture, CPU, OS selected. So this part could probably be omitted, if not necessary.

Reply:
Please remove this comment also